Documentation/Signoffs/PLC.md

```diff
@@ -0,0 +1,69 @@
+## Subsystem Function 
+
+The subsystem's role in our design is to control all electronic hardware (locks, LEDs, etc) as well as the motor rotating the platforms. It will also have a communication stream with the microcomputer to inform it when a board is checked out or restocked. 
+
+## Constraints 
+
+**The PLC:** 
+
+* **Must have at least 13 I/O pins**
```

Comment:
So the PLC will have no idea where the motor is?

Reply:
This is not acceptable. The PLC needs to control the system. How can it do that without appropriate means of communication. Revisit the constraints and method of communication to the motor system. Further, it seems the PLC can't communicate to the PC or the database. How is that going to work?